scout/server/blueprints/variants/controllers.py

```diff
@@ -1060,84 +1058,172 @@ def generate(header, lines):
     )
 
 
-def variant_export_lines(store, case_obj, variants_query):
+def variant_export_lines_common(store: MongoAdapter, variant: dict, case_obj: dict) -> list:
+    """
+    Get common variant info to be exported. Returns a list to be merged into a string
+    in suitable export format.
+    """
+    variant_line = []
+    position = variant["position"]
+    change = variant["reference"] + ">" + variant["alternative"]
+    variant_line.append(variant.get("rank_score", "N/A"))
+    variant_line.append(variant["chromosome"])
+    variant_line.append(position)
+    variant_line.append(change)
+    variant_line.append("_".join([str(position), change]))
+
+    gene_list: List[dict] = variant.get("genes", [])
+
+    if gene_list:
+        gene_info = variant_export_genes_info(store, gene_list, case_obj.get("genome_build"))
+        variant_line += gene_info
+    else:
+        empty_col = 0
+        while empty_col < 5:
+            variant_line.append(
+                "-"
+            )  # empty HGNC id, empty gene name, two empty transcripts columns, and consequence
+            empty_col += 1
+
+    if variant.get("cadd_score"):
+        variant_line.append(round(variant["cadd_score"], 2))
+    else:
+        variant_line.append("N/A")
+
+    if variant.get("gnomad_frequency"):
+        variant_line.append(round(variant["gnomad_frequency"], 5))
+    else:
+        variant_line.append("N/A")
+
+    return variant_line
+
+
+def get_fusion_variant_field(variant: dict, field: str) -> str:
+    """Get fusion variant fields that could be from multiple fusion partners.
+    Split if comma separated, and join with 'or' sign, so as not to interfere with csv file generation.
+    Avoid empty values and use "N/A" instead.
+    """
+
+    value = variant.get(field, "N/A")
+    if "," in value:
+        value = value.split(",")
+    if isinstance(value, list):
+        value = " | ".join(value)
+    if value is None or value == []:
+        value = "N/A"
+    return value
+
+
+def get_fusion_exons(variant: dict) -> str:
+    """Get RNAfusion specific variant exons for all genes/transcripts listed. Join with pipe 'or' character."""
+    exon = ""
+    for gene in variant.get("genes", []):
+        for transcript in gene.get("transcripts", []):
+            if "exon" in transcript:
+                if exon:
```

Comment:
```suggestion
```

Reply:
Hm, no, that would mean all exon strings start with " | ". This is using `bool("") == False` to check if it is the first iteration of the loop.